sound/soc/sof/sof-priv.h

```diff
@@ -406,6 +406,7 @@ struct snd_sof_dev {
 	struct sof_ipc_fw_ready fw_ready;
 	struct sof_ipc_fw_version fw_version;
 	struct sof_ipc_cc_version *cc_version;
+	u32 clk_config;
```

Comment:
what exactly is this clock configuration? Some bitmask?

Reply:
currently 0=HPRO, 1(or N)=LPRO. Not sure we might have different level of low power clock later